region/maridia/inner-green/Shaktool Room.json

```diff
@@ -82,6 +87,31 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Entry",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "clearsObstacles": ["R-Mode"]
+    },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Entry - Shaktool Done Digging",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "f_ShaktoolDoneDigging"
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "clearsObstacles": ["A", "B", "R-Mode"],
+      "devNote": "Entering the room with the flag 'f_ShaktoolDoneDigging' set means that the sand blocks and Power Bomb blocks will not spawn."
+    },
     {
       "id": 1,
       "link": [1, 1],
```

Comment:
Not sure why this needs an entrance condition. I wonder if we could just delete the entrance condition and the "R-mode done digging" one above